ui/app/common/patient-search/controllers/patientsListController.js

```diff
@@ -66,23 +66,27 @@ angular.module('bahmni.common.patientSearch')
             }
             return [];
         };
-
-        $scope.isHeadingOfIdentifier = function (heading) {
-            return _.includes(Bahmni.Common.PatientSearch.Constants.identifierHeading, heading);
+        $scope.isHeadingOfLinkColumn = function (search, heading) {
```

Comment:
search is already available in $scope. We don't need it as param.

Reply:
Yah. it's working fine with scope values. I changed it to $scope.search. thanks :)